recipe/conda_build_config.yaml

```diff
@@ -2,8 +2,3 @@ MACOSX_SDK_VERSION:        # [osx and x86_64]
   - "11.0"                 # [osx and x86_64]
 MACOSX_DEPLOYMENT_TARGET:  # [osx and x86_64]
   - "11.0"                 # [osx and x86_64]
-libevent:
-  - "2.1.10"
-  - "2.1.12"
-msgpack_c:
-  - 6
```

Comment:
This was a bot error, fixed by https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/4927

Reply:
That's right.